draft-ietf-openpgp-pqc.md

```diff
@@ -1016,10 +1026,12 @@ The algorithm-specific fields consists of:
    - The wrapped session key represented as an octet string, i.e., the output
      of the encryption procedure described in {{ecc-mlkem-encryption}}.
 
-Note that unlike most public-key algorithms, in the case of a v3 PKESK packet,
-the symmetric algorithm identifier is not encrypted.  Instead, it is prepended
-to the encrypted session key in plaintext.  In this case, the symmetric
-algorithm used MUST be AES-128, AES-192 or AES-256 (algorithm ID 7, 8 or 9).
+Note that unlike most public-key algorithms, for the ML-KEM composite schemes,
```

Comment:
Is "unlike most" really what we want to say here?  It's also true for the crypto-refresh's recommended X25519 and X448 public-key algorithms.  Maybe it's better to refer to the similiarity rather than to the difference?

Reply:
Yes, a good suggestion, done.